src/st/no1/unk_3CB34.c

```diff
@@ -0,0 +1,447 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "no1.h"
+
+// Mask for all of the statuses where the UP or DOWN button will
+// be ignored when in elevator
+//
+// Value: 0xC5CFFEDF
+// clang-format off
+#define PLAYER_STATUS_ELEVATOR_READY_MASK                                        \
```

Comment:
I'm not sure if it makes sense logically but my guess is that this is a ~mask. Like `if player & ~(MASK_TO_CHECK1  | MASK_TO_CHECK2 ...)`. That way 8 things are being checked instead of 22. This change doesn't need to be made right now but figured I should point it out.